spec/lib/api/v3/root_representer_spec.rb

```diff
@@ -29,35 +29,45 @@
 require 'spec_helper'
 
 describe ::API::V3::RootRepresenter do
+  include ::API::V3::Utilities::PathHelper
+
   let(:representer)  { described_class.new({}) }
 
   context 'generation' do
-    subject(:generated) { representer.to_json }
+    subject { representer.to_json }
 
-    describe '_links' do
-      it { is_expected.to have_json_type(Object).at_path('_links') }
+    before do
+      allow(Setting).to receive(:app_title).and_return 'Foo Project'
+    end
 
-      describe 'priorities' do
-        it { is_expected.to have_json_path('_links/priorities') }
-        it { is_expected.to have_json_path('_links/priorities/href') }
+    describe '_links' do
+      it_behaves_like 'has an untitled link' do
+        let(:link) { 'configuration' }
+        let(:href) { api_v3_paths.configuration }
       end
 
-      describe 'project' do
-        it { is_expected.to have_json_path('_links/project') }
-        it { is_expected.to have_json_path('_links/project/href') }
-        it { is_expected.to have_json_path('_links/project/templated') }
+      it_behaves_like 'has an untitled link' do
+        let(:link) { 'priorities' }
+        let(:href) { api_v3_paths.priorities }
+      end
 
-        it {
-          is_expected.to be_json_eql('/api/v3/projects/{project_id}'.to_json)
-            .at_path('_links/project/href')
-        }
-        it { is_expected.to be_json_eql(true.to_json).at_path('_links/project/templated') }
+      it_behaves_like 'has an untitled link' do
+        let(:link) { 'statuses' }
+        let(:href) { api_v3_paths.statuses }
       end
 
-      describe 'statuses' do
-        it { is_expected.to have_json_path('_links/statuses') }
-        it { is_expected.to have_json_path('_links/statuses/href') }
+      it_behaves_like 'has an untitled link' do
+        let(:link) { 'types' }
+        let(:href) { api_v3_paths.types }
       end
     end
+
+    it 'shows the name of the instance' do
+      is_expected.to be_json_eql('Foo Project'.to_json).at_path('instanceName')
+    end
+
+    it 'indicates the OpenProject version number' do
+      is_expected.to be_json_eql(Redmine::VERSION.to_semver.to_json).at_path('coreVersion')
```

Comment:
Ah ... this PR is full of nostalgia. 

You can use `OpenProject::VERSION.to_semver.to_json` just the same.

But I think it would be even better, to mock this.